Added Working with json data in postgres guide (#96)
tembo-io/website: docs/postgres_guides/working-with-json-data-in-postgres/working-with-json-data-in-postgres.md

```
 select json_each(info) FROM students;
```

<img src={JsonEach} width="500" alt="JsonEach" />

The `jsonb_set` function helps to update the value in the table following a specified condition.

```
UPDATE "students" SET "info"=jsonb_set("info"::jsonb, '{age}', '16') WHERE "info"::json->>'name'='John';
```

<img src={Update} width="900" alt="Update" />

There are many other json functions that you can use to manipulate data in the desired manner. To know about more json function, [check their official documentation](https://www.postgresql.org/docs/9.5/functions-json.html)

## Delete JSON data

You can use the `DELETE` statement to delete any specific row

```
DELETE FROM students WHERE info->>'name' = 'John';
```

## Working with Nested JSON

If a key in JSON data has an object as it’s value, we can use the combination of both `->` and `->>` operators to target that data.

```
 SELECT info->'age'->>'years' nested_age FROM students;
```

<img src={Nested} width="600" alt="Nested" />

## Conclusion

In this guide, we studied how you can work with JSON data in the Postgres database from inserting in a table to manipulating data to deleting data.

> If you are struggling to connect to Postgres on your project, we recommend you to check our well-written [Postgres guides](https://tembo.io/docs/category/postgres-guides) to solve your problem.

Reviewer: ```suggestion
Struggling to connect to Postgres on your project? Check out our other [Postgres guides](https://tembo.io/docs/category/postgres-guides) for more help.
```

Author: @ericankenman I have addressed these comments
